java/memory/src/main/java/io/netty/buffer/ArrowBuf.java

```diff
@@ -493,6 +494,16 @@ public ArrowBuf retain() {
     return retain(1);
   }
 
+  @Override
+  public ByteBuf touch() {
+    return this;
+  }
+
+  @Override
+  public ByteBuf touch(Object hint) {
+    return this;
```

Comment:
@zsxwing why does this return this, instead of invoking the super touch(Object hint)?

Reply:
AbstractByteBuf doesn't implement it.